fastlane/metadata/de-DE/release_notes.txt

```diff
@@ -1,2 +1 @@
-Mit dieser Version des Ecosia-Browsers lassen sich über den Marketplace für alternative Apps von Apple auch Apps von anderen Quellen runterladen. Dieses Feature steht Leuten mit Wohnsitz in EU-Ländern zur Verfügung.
-Wie immer wurden auch Bugs behoben und einige Verbesserungen eingearbeitet.
+
```

Comment:
I think you only need to change the source file aka the English notes. The rest will be handled by the transifex plugin.

Reply:
Yeah @ecotopian @lucaschifino what I did was following the README actually. Which meant to update the change on AppStoreConnect for English and launch the fastlane command. Maybe I'll need to update that as indeed one that needed change and I would have done as you suggested 😂.
Nice way to catch something needing amend on the README.
Both processes work either way.